sdk/ai/azure-ai-resources/pyproject.toml

```diff
@@ -7,5 +7,5 @@ pylint = false
 black = false
 strict_sphinx = false
```

Comment:
same here as in `azure-ai-generative`

Reply:
Resolved by https://github.com/Azure/azure-sdk-for-python/pull/32884/commits/5e0612fee65d65905acefb3a8a4ff43e297532a0